graphics/include/ignition/common/MeshLoader.hh

```diff
@@ -32,7 +33,9 @@ namespace ignition
     class IGNITION_COMMON_GRAPHICS_VISIBLE MeshLoader
     {
       /// \brief Constructor
-      public: MeshLoader() = default;
+      public: MeshLoader() {
+        std::setlocale(LC_NUMERIC, "C");
```

Comment:
This is influencing the global state of the process in a way that is not documented and may not be ideal for a library. Furthermore, other part of the process can then set again the `LC_NUMERIC` to other values, breaking the functionality of the class **after** it has been created.

Reply:
I think that this is relevant to what you explained here https://github.com/ignitionrobotics/ign-rendering/issues/136, right? So instead of this PR we should try to identify which part of the code is using locale dependent commands and apply changes there.

Reply:
Exactly. To be perfectly clear, it is not tragic if this is merged, but I still wanted to raise the point that is not ideal to perturb the global state from a library.

Reply:
I agree, this is kind of a concerning workaround.  It may not break anything as-is, but it's sort of a bad "code smell" that this is necessary.

Reply:
As discussed in one of the meetings, let's decline this PR and try to address the underlying issue instead.